code/modules/jobs/job_types/wasteland.dm

```diff
@@ -64,7 +71,7 @@ wasteland
 	backpack_contents = list(
 		/obj/item/reagent_containers/hypospray/medipen/stimpak=2, \
 		/obj/item/reagent_containers/blood/radaway=1, \
-		/obj/item/reagent_containers/food/drinks/flask, \
```

Comment:
Leave the flask in. Maybe randomspawn what alcohol or water starts in it.

Reply:
They spawn with two, nappist. The second is universally thrown out.